SPECS/python-certifi/python-certifi.spec

```diff
@@ -1,51 +1,67 @@
 Summary:        Python package for providing Mozilla's CA Bundle
 Name:           python-certifi
-Version:        2022.12.07
+Version:        2023.05.07
 Release:        1%{?dist}
 License:        MPL-2.0
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
-Group:          Development/Languages/Python
-URL:            https://github.com/certifi
-#Source0:       https://github.com/certifi/python-certifi/archive/refs/tags/%{version}.tar.gz
-Source0:        certifi-%{version}.tar.gz
-BuildArch:      noarch
+URL:            https://certifi.io/
+Source:         https://github.com/certifi/%{name}/archive/%{version}/%{name}-%{version}.tar.gz
+Patch0:         certifi-2022.12.07-use-system-cert.patch
 
-%description
-Python package for providing Mozilla's CA Bundle
+BuildArch:      noarch
 
-%package -n     python3-certifi
-Summary:        Python package for providing Mozilla's CA Bundle
 BuildRequires:  python3-devel
-BuildRequires:  python3-setuptools
-%if %{with_check}
 BuildRequires:  python3-pip
+BuildRequires:  python3-wheel
+
+%if %{with_check}
+BuildRequires:  ca-certificates-base
```

Comment:
Consider making this an explicit requires with or without check.

Reply:
I was considering it, but I don't want to hard-code a dependency on a specific package `ca-certificates` package, as `ca-certificates` and `ca-certificates-base` (and any bundles we might thing of to provide in the future) are designed to be installable independently of each other. For instance you case save some space, if you install `ca-certificates` and remove `ca-certificates-base` without.

Having said that, I think we can work with a run-time dependency on the path to the certificates bundle itself.